extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -449,6 +460,8 @@ export class AmpA4A extends AMP.BaseElement {
       this.a4aAnalyticsElement_ = insertAnalyticsElement(
           this.element, this.a4aAnalyticsConfig_, true /* loadAnalytics */);
     }
+
+    this.isSinglePageStoryAd = 'ampStory' in this.element.dataset;
```

Comment:
Right now this is not `data-` prefixed so I don't think this check will work. `this.element.hasAttribute('amp-story')`

Reply:
Fixed.